config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  // This origin trial token is used to enable WebVR and Gamepad Extensions on Chrome 62+
+  // You can find more information about getting your own origin trial token here: https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md
+  originTrialToken:
+    "AvIMoF4hyRZQVfSfksoqP+7qzwa4FSBzHRHvUyzC8rMATJVRbcOiLewBxbXtJVyV3N62gsZv7PoSNtDqqtjzYAcAAABkeyJvcmlnaW4iOiJodHRwczovL3JldGljdWx1bS5pbzo0NDMiLCJmZWF0dXJlIjoiV2ViVlIxLjFNNjIiLCJleHBpcnkiOjE1MTYxNDYyMDQsImlzU3ViZG9tYWluIjp0cnVlfQ==",
+  originTrialExpires: "2018-01-16",
+
+  // These variables are availible on the window.GLOBAL object
+  // The exported object must be a valid argument to JSON.stringify()
+  global: {
```

Comment:
Why do this global thing instead of just having the webpack config import config.js from its previous location?

Reply:
Good point. I was thinking that all the per build config should be in one place, but now that I'm looking at it I think a bigger reason is `process.env.JANUS_SERVER` isn't available in the browser context so I'd assume we'd want to keep it out of the `/src` directory.

I could move everything but the janus server back to the original config file though. Would that be better?